scripts/deploy.sh

```
@@ -1,32 +1,72 @@
+        break
+      else
+        echo "Invalid selection. Please try again."
+      fi
+    done
+  fi
+}
 
 # 最新のバージョン番号を取得
-LATEST_VERSION=$(clasp versions | grep -oP '^\d+' | head -n 1)
+get_latest_version() {
+  local versions=$(clasp versions | grep -oP '^\d+')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)